src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/custom.py

```diff
@@ -450,7 +450,7 @@ def acs_create(resource_group_name, deployment_name, name, ssh_key_value, dns_na
     :type master_storage_profile: str
     :param agent_profiles: AgentPoolProfiles used to describe agent pools
     :type agent_profiles: dict
-    :param agent_count: The number of agents for the cluster.  Note, for
+    :param agent_count: Set default number of agents for the agent pools.  Note, for
```

Comment:
The note should arguably go in the `help.py` file as a `long-summary` entry.

Reply:
I moved the long version to _params.py file. I am very clear the strategy's here to use `long-summary`, after i get familiarized, i probably need to do that across the code.